sdk/react/panels/style/languageSelectionPanelStyles.json

```diff
@@ -55,6 +55,10 @@
     "alignSelf": "stretch",
     "backgroundColor": "#585858"
   },
+  "selectionPreviewtextContainer": {
```

Comment:
Follow camel case for all the words, it should be `selectionPreviewTextContainer`. Can it be simpler? e.g. `previewTextContainer`